build-system/pr-check/validator-tests.js

```diff
@@ -36,18 +36,20 @@ const timedExecOrDie =
 function main() {
   const startTime = startTimer(FILENAME, FILENAME);
   const buildTargets = determineBuildTargets();
-  printChangeSummary(FILENAME);
 
   if (!isTravisPullRequestBuild()) {
     timedExecOrDie('gulp validator');
     timedExecOrDie('gulp validator-webui');
-  } else if (buildTargets.has('VALIDATOR')) {
-    timedExecOrDie('gulp validator');
-  } else if (buildTargets.has('VALIDATOR_WEBUI')) {
-    timedExecOrDie('gulp validator-webui');
   } else {
-    console.log('Skipping validator job because this commit does ' +
-      'not affect the validator or validator web UI.');
+    printChangeSummary(FILENAME);
+    if (buildTargets.has('VALIDATOR')) {
+      timedExecOrDie('gulp validator');
+    } else if (buildTargets.has('VALIDATOR_WEBUI')) {
+      timedExecOrDie('gulp validator-webui');
+    } else {
+      console.log('Skipping validator job because this commit does ' +
+        'not affect the validator or validator web UI.');
```

Comment:
While you're at it, add `fileLogPrefix` to this logging statement.

Reply:
Done for this & other jobs when they are skipped